src/App.css

```diff
@@ -1,6 +1,7 @@
 :root {
   --main-yellow: #f8ed44;
   --main-grey: #878680;
+  --dark-grey: #1F1F1F;
```

Comment:
Can we eventually change this to main-dark-grey since it's our our new site grey we'll used on the site? Not sure where the original main-grey itself is used, but would be great for when we do a styling refactor.